isaric/parsers/isaric-ccpuk.toml

```diff
@@ -961,7 +975,28 @@
   context = ["NEW symptom"]
```

Comment:
??.. what is this in relation to? or how is this recorded? new during admission / study period?

Reply:
The data dict says the followup symptoms should only be recorded if they're new since the last survey - thought it should be a context note.